src/main/resources/library/attribute-definitions.json

```diff
@@ -43,9 +45,18 @@
       "items": { "type": "string" },
       "title": "Research Institute"
     },
+    "library:diseaseOntologyDOID": {
+      "type": "string",
+      "hidden": true
+    },
+    "library:diseaseOntology": {
+      "type": "string",
+      "title": "Cohort Phenotypic Inclusion Criteria (Disease Ontology)",
+      "typeahead": "ontology"
+    },
```

Comment:
what is the value of typeahead used for? (just thinking about when will do autocomplete for ES)